es-design-system/pages/organisms/es-support-card.vue

```diff
@@ -0,0 +1,369 @@
+                    </dl>
+                    <dl>
+                        <dt>
+                            Default
+                        </dt>
+                        <dd>
+                            <code>''</code>
+                        </dd>
+                    </dl>
+                    <dl>
+                        <dt>
+                            Description
+                        </dt>
+                        <dd>
+                            The URL to which the secondary CTA button (if any) should link.
+                        </dd>
+                    </dl>
+                </ds-responsive-table-row>
+            </ds-responsive-table>
+        </div>
+        <div class="mb-450">
+            <h2>
+                Slots
+            </h2>
+            <ds-responsive-table>
+                <ds-responsive-table-row>
+                    <dl>
+                        <dt>
+                            Name
+                        </dt>
+                        <dd>
+                            <code>headline</code>
+                        </dd>
+                    </dl>
+                    <dl>
+                        <dt>
+                            Default
+                        </dt>
+                        <dd>
+                            n/a
+                        </dd>
+                    </dl>
+                    <dl>
+                        <dt>
+                            Description
+                        </dt>
+                        <dd>
+                            The content to insert into the card's heading.
+                        </dd>
+                    </dl>
+                </ds-responsive-table-row>
+                <ds-responsive-table-row>
+                    <dl>
+                        <dt>
+                            Name
+                        </dt>
+                        <dd>
+                            <code>description</code>
+                        </dd>
+                    </dl>
+                    <dl>
+                        <dt>
+                            Default
+                        </dt>
+                        <dd>
+                            n/a
+                        </dd>
+                    </dl>
+                    <dl>
+                        <dt>
+                            Description
+                        </dt>
+                        <dd>
+                            The content to display in the card's main paragraph.
+                        </dd>
+                    </dl>
+                </ds-responsive-table-row>
+            </ds-responsive-table>
+        </div>
+        <ds-doc-source
+            :comp-code="compCode"
+            comp-source="es-vue-base/src/lib-components/EsSupportCard.vue"
+            :doc-code="docCode"
+            doc-source="es-design-system/pages/organisms/es-support-card.vue" />
+    </div>
+</template>
+
+<script>
+import eaMultiple from '@/assets/img/ea-multiple.png';
```

Comment:
Instead of doing this, can we use something like https://place-hold.it/?

Reply:
Changed to use the IconPerson SVG, but as an imported image asset, so we can pass it into image `src` props.